queue/queueservice.js

```diff
@@ -468,7 +486,7 @@ class QueueService {
     }
 
     // Put message queue
-    return this._putMessage(queueName, {
+    return this._putMessage(queueNames[task.priority], {
```

Comment:
isn't queueNames:

```js
{
  <provisionerId>/<workerType>: [<normal priority>, <high priority>]
}
```
If so, the keys in queueNames wouldn't be task.priority... unless I'm misunderstanding something